libs/platform/src/lib/components/value-help-dialog/value-help-dialog/value-help-dialog.component.html

```diff
@@ -276,20 +276,32 @@ <h5 fd-title>{{ searchField.advancedSearchLabel }}</h5>
 </ng-template>
 
 <ng-template #mobileTabs>
-    <ul fd-list [compact]="true" *ngIf="!hasSelectedTab && showSelectionTab && showDefineTab">
-        <li fd-list-item *ngIf="showSelectionTab">
-            <a fd-list-link (click)="switchTab(_tabTypes.selectFromList)">
-                <span fd-list-title>{{ selectTabTitle }}</span>
-                <i fd-list-icon glyph="navigation-right-arrow"></i>
-            </a>
-        </li>
-        <li fd-list-item *ngIf="showDefineTab">
-            <a fd-list-link (click)="switchTab(_tabTypes.defineConditions)">
-                <span fd-list-title>{{ defineTabTitle }}</span>
-                <i fd-list-icon glyph="navigation-right-arrow"></i>
-            </a>
-        </li>
-    </ul>
+  <div *ngIf="!hasSelectedTab && showSelectionTab && showDefineTab">
+    <div fd-bar [inPage]="true"  *ngIf="showSelectionTab">
+      <div fd-bar-left>
+        <fd-bar-element>
+            {{ selectTabTitle }}
+        </fd-bar-element>
+      </div>
+    <div fd-bar-right>
+        <fd-bar-element>
+			<fd-button-bar [glyph]="'navigation-right-arrow'" (click)="switchTab(_tabTypes.selectFromList)"></fd-button-bar>
+        </fd-bar-element>
+    </div>
+  </div>
+  <div fd-bar [inPage]="true" *ngIf="showDefineTab">
+    <div fd-bar-left>
+        <fd-bar-element>
+            {{ defineTabTitle }}
+        </fd-bar-element>
+    </div>
+    <div fd-bar-right>
+       <fd-bar-element>
+		  <fd-button-bar [glyph]="'navigation-right-arrow'" (click)="switchTab(_tabTypes.defineConditions)"></fd-button-bar>
```

Comment:
unnecessary binding. same in [line 288](https://github.com/SAP/fundamental-ngx/pull/5810/files#diff-6916bbb15085a5ee4f6545073ce063e440e94cddbab4188009abf452de368a30R288)

Reply:
fixed